benchmarks/pom.xml

```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>io.zipkin</groupId>
     <artifactId>zipkin-parent</artifactId>
-    <version>3.3.1-SNAPSHOT</version>
+    <version>3.4.0-SNAPSHOT</version>
```

Comment:
I don't think we should bump minor on this, rather bump minor afterwards when you add the elastic stuff.

Reply:
so after we bump all the deps, we cut 3.3.1, and after that's out we bump your PR for 3.4.0

Reply:
Got it, sure, I will revert that